schwifty/bank_registry/manual_pt.json

```diff
@@ -0,0 +1,170 @@
+[
+    {
+        "country_code": "PT",
+        "primary": true,
+        "bic": "ACTVPTPL",
+        "bank_code": "ACTV",
```

Comment:
Also for Portuguese banks, the domestic bank code is in fact numeric. It has to match the bank code part of the IBAN or otherwise schwifty wouldn't be able to find the BIC related to a given IBAN.
https://wise.com/gb/iban/portugal

Reply:
amended the commit, with the correct bank codes, feel free to have a look